src/BenchmarksDriver2/benchmarks.proxy.yml

```diff
@@ -5,11 +5,12 @@
 # --config benchmarks.proxy.yml --scenario proxy-envoy --profile aspnet-physical
 
 imports:
-  - https://raw.githubusercontent.com/aspnet/Benchmarks/master/src/Wrk2Client/wrk2.yml
   - https://raw.githubusercontent.com/aspnet/Benchmarks/master/src/BombardierClient/bombardier.yml
 
 variables:
     path: /
+    serverScheme: http
```

Comment:
There should be a file in /scenarios that is used now by the CI and documented. This file is obsolete. We can add a comment at the top to mention that.